docs/design/proposals/encrypted-pvc.md

```diff
@@ -127,3 +127,10 @@ data:
 metadata:
   name: ceph-csi-encryption-kms-config
 ```
+
+NOTE:
+
+KMS vault can be authenticated using token auth method or kubernetes auth
```

Comment:
Vauls and Kubernetes are names, use **V**ault and **K**ubernetes. Maybe even **K**ubernetes **A**uth